cmd/ctr/commands/images/pull.go

```diff
@@ -120,16 +120,17 @@ command. As part of this process, we do the following:
 				return err
 			}
 
-			// Set unpack configuration
-			for _, platform := range p {
-				sopts = append(sopts, image.WithUnpack(platform, cliContext.String("snapshotter")))
-			}
 			if !cliContext.Bool("all-platforms") {
 				if len(p) == 0 {
 					p = append(p, platforms.DefaultSpec())
 				}
 				sopts = append(sopts, image.WithPlatforms(p...))
 			}
+
+			// Set unpack configuration
```

Comment:
Why did you move this section?

Reply:
(reverted to main now )

My fix was the following: https://github.com/containerd/containerd/commit/79a42eedc724cd248a995cbf1174d3800d948d52

I wasn't understanding why my pull was so fast, and why in kube so slow, turns out a pull was not unpacking by default.

Someone else merged a fix before me, so, reverted.